cmd/spire-server/cli/run/run.go

```diff
@@ -695,6 +751,23 @@ func parseBundleEndpointProfile(config federatesWithConfig) (trustDomainConfig *
 	}, nil
 }
 
+func parseSingleAstNode(node ast.Node) (string, error) {
```

Comment:
We may consider renaming this function since it has logic specifically for bundle endpoint profile handling e.g. `parseBundleEndpointProfileASTNode` or maybe something shorter 😂